### DISCUSSION TITLE
Why assert False don't call pytest_assertrepr_compare

### DISCUSSION THREAD

**Discussion by lozik4** [pytest-dev/pytest#11791] [Q&A]
I can rewrite assert message using _**pytest_assertrepr_compare**_
if my assert message has view `assert False is True` pytest call pytest_assertrepr_compare and everything works
But if use syntax `assert False` pytest_assertrepr_compare don't call.
In docs I didn't see this information [https://docs.pytest.org/en/7.1.x/reference/reference.html?highlight=pytest_assertrepr_compare#std-hook-pytest_assertrepr_compare](url)

**Comment by RonnyPfannschmidt** [comment]
The repr compare hook gets called for actual operators between 2 values 

False on it's own is only a value

**Reply by lozik4** [reply]
But _**assert False**_ print standard pytest trace
